packages/client/src/layers/phaser/systems/createMapSystem.ts

```diff
@@ -37,6 +38,8 @@ export function createMapSystem(context: PhaserLayer, network: NetworkLayer) {
   chunks.addedChunks$.subscribe(addedChunks$);
   const newTile$ = new Subject<Coord>();
 
+  // Prevent redrawing the entire map on each tick
+  (phaserScene.game.renderer.config as any).clearBeforeRender = false;
```

Comment:
To me, this reads like it'll still render on each tick, it just won't clear the canvas before doing so, potentially leaving artifacts of the previous render. Did you play with this to find out how this works? The docs aren't super clear.

Reply:
that's true, the comment is wrong. It does skip the step of clearing the map before each redraw, which should be fine in our case because the map doesn't change